apps/SageAccountWeb/package.json

```diff
@@ -23,7 +23,7 @@
     "react-easy-crop": "^5.0.7",
     "react-router-dom": "^5.3.4",
     "sass": "^1.71.1",
-    "synapse-client": "workspace:*",
+    "synapse-client": "workspace:@sage-bionetworks/synapse-client@*",
```

Comment:
With this alias we can import using the shorter `'synapse-client'` while still publishing under the @sage-bionetworks namespace